instrumentation/opentelemetry-instrumentation-aiopg/src/opentelemetry/instrumentation/aiopg/wrappers.py

```diff
@@ -23,7 +23,7 @@
 
     from opentelemetry import trace
     from opentelemetry.instrumentation.aiopg import trace_integration
-    from opentelemetry.trace import TracerProvider
+    from opentelemetry.sdk.trace import TracerProvider
```

Comment:
As mentioned in the thread, please just remove the bootstrapping code for the TracerProvider period. 

@lzchen's point is accurate that we should not reference the SDK explicitly for examples or in it's operation itself.

For clarity: the tests are an exception because we need some concrete implementation.

Reply:
Hi @toumorokoshi, Thanks for the suggestions. The usage examples such as [boto](https://opentelemetry-python.readthedocs.io/en/stable/instrumentation/boto/boto.html) has referenced the SDK. I took that as an inspiration to make these changes. Would you recommend me to remove the `TracerProvider` there also?

Reply:
I think it's the lesser of two evils, in this regard. I'm +1 for removing this boilerplate across all repos (sdk or not). 

Let's get @lzchen or another approver/maintainer perspective before moving forward.

Reply:
I agree it is an antipattern. Although from a customer's point of view, I would go to the actual instrumentation package of interest and use whatever examples are available directly usually and just expect them to work. Do you think this is because we don't have a very explicit "USE THIS AS ACTUAL CODE" AND "IN-YOUR-FACE" getting started? Our current Getting Started also does not have anything related to setting up a tracing pipeline so maybe we should try adding that first and seeing how the response is. @codeboten

Reply:
> Our current Getting Started also does not have anything related to setting up a tracing pipeline

Does [this](https://opentelemetry-python.readthedocs.io/en/stable/getting-started.html#configure-exporters-to-emit-spans-elsewhere) not do the trick? If you follow the first two sections you will have traces emitting to stdout.

If you follow the first three sections you should have enough knowledge to understand the instrumentation boilerplate.

> I would go to the actual instrumentation package of interest and use whatever examples are available directly usually and just expect them to work. 

I'd agree with that as well, although I would say most instrumentation is not useful until you're exporting it somewhere besides stdout. And we don't know what exporter makes sense for you as a user (could be jaeger, could be zipkin).

Reply:
> The usage examples such as boto has referenced the SDK.

@lonewolf3739 
Yes the usage section for each instrumentation should only be showing how to instrument [see flask](https://github.com/open-telemetry/opentelemetry-python/blob/master/instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py#L32).

@toumorokoshi 
> If you follow the first three sections you should have enough knowledge to understand the instrumentation boilerplate.

Right I think I was referring to one whole section with trace setup -> instrumentation -> exporter pipeline, but the `Integrations with Flask` section should be sufficient.